include/api.php

```diff
@@ -317,9 +338,10 @@ function pll_set_post_language( $id, $lang ) {
  *
  * @api
  * @since 1.5
+ * @since 3.4 $lang accepts PLL_Language, string, or int.
```

Comment:
Same as above.

Reply:
Done in [10ec660](https://github.com/polylang/polylang/pull/1149/commits/10ec660bc67a078dde283dd1c86b5566cee60343).